src/applications/vaos/new-appointment/redux/actions.js

```diff
@@ -290,7 +290,7 @@ export function fetchFacilityDetails(facilityId) {
   };
 }
 
-export function checkEligibility(location, siteId) {
+export function checkEligibility({ location, siteId, showModal }) {
```

Comment:
Add this `showModal` flag so we can prevent it from showing if we return to the page and do an eligibility check.